src/index-metadbs/2dmatpedia/v1/links.json

```diff
@@ -0,0 +1,34 @@
+{
+   "meta" : {
+       "api_version" : "1.0.0",
+       "query" : {
+           "representation" : "/links"
+       },
+       "more_data_available" : false,
+       "schema" : "https://schemas.optimade.org/openapi/v1.0/optimade_index.json"
+   },
+   "data" : [
+      {
+         "id" : "2dmatpedia",
```

Comment:
In the database you are using the prefix `twodm` instead - will you want to keep `2dmatpedia` or `twodm` (either updating to `twodm` here or to `2dmatpedia` in your implementation at http://optimade.2dmatpedia.org)?